src/Admin/Model/UserAdmin.php

```diff
@@ -20,13 +20,30 @@
 use Sonata\AdminBundle\Form\FormMapper;
 use Sonata\AdminBundle\Show\ShowMapper;
 use Sonata\UserBundle\Form\Type\SecurityRolesType;
+use Sonata\UserBundle\Model\UserManagerInterface;
 use Symfony\Component\Form\Extension\Core\Type\TextType;
 
 /**
  * @phpstan-extends AbstractAdmin<\Sonata\UserBundle\Model\UserInterface>
  */
 class UserAdmin extends AbstractAdmin
 {
+    private ?UserManagerInterface $userManager;
```

Comment:
Shouldn't we provide a getter or make the property protected if the class is not final ?

Reply:
I made it protected.